### PR TITLE
adding preliminary meta-data format specification

### PR DESCRIPTION
**Pull request by PapillonMcGill**
adding meta-data based on bioCADDIE DATS format.

### CODE REVIEW

**Comment by glatard** [comment]
Great, thanks for this!

**Comment by jbpoline** [comment]
Sounds like a good start !

One question: once in BIDS, a bunch of metadata will be available in DataLad metadata, so we need to see how these could be incorporated / linked -any idea on this ? 

Also, could you try to get something like PreventAD with Jennifer (I'll send you a mail to introduce you two) ? having a couple of examples would be good to get a better idea how this will work.

File: README.md [hunk 1]
```diff
@@ -146,6 +146,35 @@ decide on the creation of sub-datasets in it. Modifications to
...
+Here is an example:
+```
+{
+    "schema": "https://github.com/biocaddie/DATS/blob/master/dataset_schema.json",
+    "title": "SDas Phantom",
+    "description": "Samir Das phantom scans",
+    "dates": [ "2018-12-05" ],
+    "creators": [ "McGill Centre for Integrative Neuroscience" ],
+    "storedIn": "",
+    "type": [ "fMRI", "phantom" ],
```
Comment (jbpoline) [inline README.md]: I may be mistaking but could it be (according to the json schemas) something like :
 "types": [
    "information": [
            {"value": "fMRI", 
            "valueIRI":  http://uri.interlex.org/base/ilx_0506130 },
            {"value":"phantom",
             "valueIRI": ""}
           ]
    ]

Comment (jbpoline) [reply README.md]: see this example

"types": [
    {
      "information": {
        "value": "Survey",
        "valueIRI" : ""
      }
    },
    {
      "information": {
        "value": "Behavioral",
        "valueIRI" : ""
      }
    },
    {
      "information": {
        "value": "Administrative",
        "valueIRI" : ""
      }
    }
],

**Comment by PapillonMcGill** [comment]
@jbpoline Yes, you are right, I submitted a new PR as this one is already merged.

**Pull request merged**